projects/plugins/jetpack/modules/sso.php

```diff
@@ -320,10 +371,10 @@ public function render_require_two_step() {
 			<input
 				type="checkbox"
 				name="jetpack_sso_require_two_step"
-				<?php checked( Jetpack_SSO_Helpers::is_two_step_required() ); ?>
-				<?php disabled( Jetpack_SSO_Helpers::is_require_two_step_checkbox_disabled() ); ?>
+		<?php checked( Jetpack_SSO_Helpers::is_two_step_required() ); ?>
+		<?php disabled( Jetpack_SSO_Helpers::is_require_two_step_checkbox_disabled() ); ?>
 			>
-			<?php esc_html_e( 'Require Two-Step Authentication', 'jetpack' ); ?>
+		<?php esc_html_e( 'Require Two-Step Authentication', 'jetpack' ); ?>
```

Comment:
Is there a reason these were pulled back, it's fine if it was automated formatting.

Reply:
It was automated formatting, I believe the previous version was unformatted